### ISSUE TITLE
Add docs

### ISSUE THREAD

**Issue by neothemachine** [Turbo87/sidebar-v2#50]
There should be some documentation.

One trivia I already found: The sidebar HTML has to come *before* the leaflet map div, otherwise it overlays the zoom buttons. Maybe that's also a bug?

**Comment by Turbo87** [comment]
You are right, we certainly need better docs. Unfortunately I haven't found the time yet to write them. Feel free to help me if you want to.

**Comment by woodbri** [comment]
What is the correct way to programmatically open one of the sidebar tabs? I want to have an info tab that get populated via a click and ajax request and then open the that tab.

**Comment by Turbo87** [comment]
@woodbri most/all of the implementations have an `open()` method with an ID parameter of the tab you want to open. e.g. https://github.com/Turbo87/sidebar-v2/blob/b906dd6885218a0b86fcecb4453e8a48dbe3cbf2/js/leaflet-sidebar.js#L131

**Comment by woodbri** [comment]
I need to some events to trigger when the sidebar opens and collapses? How can I get these?
